lib/pingCodeOwnerAfter24hrs.js

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Rename this file to exclude the 24 hours, so something like pingCodeOwners.js

Reply:
```suggestion
// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Reply:
On it Sir.